src/index.ts

```diff
@@ -53,15 +53,15 @@ function getInstance(): SignInWithAuth0 {
 }
 
 export const initAuth0: InitAuth0 = (params) => {
-  const config = getConfig(getParams(params));
+  const config = getConfig(getParams(params)) as Config;
```

Comment:
This is related to coupling this config object to the session package's one. As we're extending that config object, that should not be an issue because it still contains all the properties from the session package config.